### PR TITLE
Make SchildiChat Web/Desktop actually appear under clients

### PR DESCRIPTION
**Pull request by su-ex**
Till now it's missing although added.

### CODE REVIEW

**Review by MadLittleMods — CHANGES_REQUESTED** [review]
Looks good once we adjust the `sort_order`

http://localhost:8000/clients/
Screen Shot 2021-06-09 at 7 00 41 PM


http://localhost:8000/docs/projects/try-matrix-now/
Screen Shot 2021-06-09 at 6 59 33 PM

http://localhost:8000/docs/projects/client/schildichat
Screen Shot 2021-06-09 at 7 01 12 PM

File: gatsby/content/projects/clients/schildichat-desktop.mdx [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
+room: "#schildichat-web:matrix.org"
+SDK: matrix-js-sdk, matrix-react-sdk
 platforms:
-    - Windows
     - Linux
     - Mac
+    - Windows
     - Web
+featured: true
+sort_order: 100
```
[the remark this answers is not in the document]
Comment (su-ex) [reply gatsby/content/projects/clients/schildichat-desktop.mdx]: Done. Although some of the other clients have it too.
Didn't know where to put it so left this to others to adjust. Maybe alphabetic ordering would be fair or make it random each build. 🎲😆

[the remark this answers is not in the document]
Comment (MadLittleMods) [reply gatsby/content/projects/clients/schildichat-desktop.mdx]: > Although some of the other clients have it too.

I think I am going to be creating a pull request to remove it from all of the other clients too.

**Comment by MadLittleMods** [comment]
Thanks for the updates @su-ex 🐸 merged 🚀

**Comment by su-ex** [comment]
Thanks!

**Pull request merged**